shuup/gdpr/static_src/gdpr.js

```diff
@@ -35,7 +35,19 @@
     });
 
     function saveConsent() {
-        $("#consent-form").submit();
+        data = $("#consent-form").serialize();
+        var request = $.ajax({
+            url: $("#consent-form").attr("action"),
+            type: 'POST',
+            data: data,
+            success: function() {
+                var consent_bar = document.getElementsByClassName("gdpr-consent-warn-bar")[0];
```

Comment:
remove the element using just jQuery that won't trow error if the elements doesn't exist anymore:
```js
$(".gdpr-consent-warn-bar").remove();
$(".gdpr-consent-preferences").remove();
```